<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pokefinder.ico</normaloff>../pokefinder.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tabGen3">
       <attribute name="title">
        <string>Gen 3</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonEgg3">
          <property name="text">
           <string>Egg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonGameCube">
          <property name="text">
           <string>GameCube</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonIDs3">
          <property name="text">
           <string>IDs</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonStatic3">
          <property name="text">
           <string>Static</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButtonWild3">
          <property name="text">
           <string>Wild</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGen4">
       <attribute name="title">
        <string>Gen 4</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonEgg4">
          <property name="text">
           <string>Egg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonEvent4">
          <property name="text">
           <string>Event</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonIDs4">
          <property name="text">
           <string>IDs</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonStatic4">
          <property name="text">
           <string>Static</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButtonWild4">
          <property name="text">
           <string>Wild</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGen5">
       <attribute name="title">
        <string>Gen 5</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonDreamRadar">
          <property name="text">
           <string>Dream Radar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonEgg5">
          <property name="text">
           <string>Egg</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonEvent5">
          <property name="text">
           <string>Event</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonHiddenGrotto">
          <property name="text">
           <string>Hidden Grotto</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButtonIDs5">
          <property name="text">
           <string>IDs</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButtonStatic5">
          <property name="text">
           <string>Static</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButtonWild5">
          <property name="text">
           <string>Wild</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGen8">
       <attribute name="title">
        <string>Gen 8</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonEgg8">
          <property name="text">
           <string>Egg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonEvent8">
          <property name="text">
           <string>Event</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonIDs8">
          <property name="text">
           <string>IDs</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonRaid">
          <property name="text">
           <string>Raid</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButtonStatic8">
          <property name="text">
           <string>Static</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButtonUnderground">
          <property name="text">
           <string>Underground</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButtonWild8">
          <property name="text">
           <string>Wild</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionEncounterLookup"/>
    <addaction name="actionIVCalculator"/>
    <addaction name="actionResearcher"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuGen3Tools">
    <property name="title">
     <string>Gen 3 Tools</string>
    </property>
    <addaction name="actionGameCubeSeedFinder"/>
    <addaction name="actionIVstoPID3"/>
    <addaction name="actionPIDtoIVs"/>
    <addaction name="actionPokeSpot"/>
    <addaction name="actionProfileManager3"/>
    <addaction name="actionSeedToTime3"/>
    <addaction name="actionSpindaPainter"/>
   </widget>
   <widget class="QMenu" name="menuGen4Tools">
    <property name="title">
     <string>Gen 4 Tools</string>
    </property>
    <addaction name="actionIVstoPID4"/>
    <addaction name="actionProfileManager4"/>
    <addaction name="actionSeedToTime4"/>
    <addaction name="actionSIDfromChainedShiny"/>
   </widget>
   <widget class="QMenu" name="menuGen5Tools">
    <property name="title">
     <string>Gen 5 Tools</string>
    </property>
    <addaction name="actionProfileCalibrator"/>
    <addaction name="actionProfileManager5"/>
   </widget>
   <widget class="QMenu" name="menuGen8Tools">
    <property name="title">
     <string>Gen 8 Tools</string>
    </property>
    <addaction name="actionDenMap"/>
    <addaction name="actionProfileManager8"/>
   </widget>
   <addaction name="menuGen3Tools"/>
   <addaction name="menuGen4Tools"/>
   <addaction name="menuGen5Tools"/>
   <addaction name="menuGen8Tools"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionResearcher">
   <property name="text">
    <string>Researcher</string>
   </property>
  </action>
  <action name="actionPokeSpot">
   <property name="text">
    <string>Poke Spot</string>
   </property>
  </action>
  <action name="actionSeedToTime4">
   <property name="text">
    <string>Seed to Time</string>
   </property>
  </action>
  <action name="actionSIDfromChainedShiny">
   <property name="text">
    <string>SID from Chained Shiny</string>
   </property>
  </action>
  <action name="actionIVCalculator">
   <property name="text">
    <string>IV Calculator</string>
   </property>
  </action>
  <action name="actionSpindaPainter">
   <property name="text">
    <string>Spinda Painter</string>
   </property>
  </action>
  <action name="actionEncounterLookup">
   <property name="text">
    <string>Encounter Lookup</string>
   </property>
  </action>
  <action name="actionGameCubeSeedFinder">
   <property name="text">
    <string>GameCube Seed Finder</string>
   </property>
  </action>
  <action name="actionProfileCalibrator">
   <property name="text">
    <string>Profile Calibrator</string>
   </property>
  </action>
  <action name="actionProfileManager3">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionProfileManager4">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionProfileManager5">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDenMap">
   <property name="text">
    <string>Den Map</string>
   </property>
  </action>
  <action name="actionProfileManager8">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionPIDtoIVs">
   <property name="text">
    <string>PID to IVs</string>
   </property>
  </action>
  <action name="actionIVstoPID3">
   <property name="text">
    <string>IVs to PID</string>
   </property>
  </action>
  <action name="actionIVstoPID4">
   <property name="text">
    <string>IVs to PID</string>
   </property>
  </action>
  <action name="actionSeedToTime3">
   <property name="text">
    <string>Seed to Time</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
